src/BenchmarkDotNet/Configs/WakeLockType.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace BenchmarkDotNet.Configs
+{
+    public enum WakeLockType
+    {
+        /// <summary>
+        /// Forces the system to be in the working state while benchmarks are running.
+        /// </summary>
+        RequireSystem,
+
+        /// <summary>
+        /// Allows the system to enter sleep and/or turn off the display while benchmarks are running.
+        /// </summary>
+        None,
```

Comment:
I know we chose to have system be the default, but I think `None` should be the 0 enum value, and these be in their order of precedence.

Reply:
System be default means that None must 'beat' system and display must 'beat' none, so that combining the different config values result in the 'highest' value. I was lazy and reordered the enum members. Indeed, making this ordering in ManualConfig.GetWakeLock explicit results in better readable and maintanable code.